tests/integration/test_tket.py

```diff
@@ -25,7 +25,9 @@
 tket2_installed = find_spec("tket2") is not None
 
 
-@pytest.mark.skipif(not tket2_installed, reason="Tket2 is not installed")
+# TODO: Remove this skip once https://github.com/CQCL/hugr/issues/1424 is fixed
```

Comment:
The bug from https://github.com/CQCL/hugr/issues/1424 causes tket2 to not recognise the float consts

Reply:
Fixed in https://github.com/CQCL/hugr/pull/1427